src/item.h

```diff
@@ -111,7 +111,13 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
         item &operator=( const item & ) = default;
         virtual ~item() = default;
 
-        item( const std::string new_type, int turn, bool rand = true );
```

Comment:
Are you *sure* you got all invocations of this constructor?
In this case it probably won't be as dangerous as my adventures with `map` class involving `int, int` to `tripoint` conversions, but you could add a
```
item( const std::string, int, bool ) = delete;
```

Just to be sure.

Unless you've got some other method, like static code analysis tool or something like that.